conf/application.conf

```diff
@@ -3,7 +3,8 @@ play.http.port=9000
 
 auth.url="http://localhost:8080"
 auth.callback="http://localhost:9000/callback"
-auth.secret="set_me"
+auth.secret="secretValue"
+auth.secret=${?AUTH_SECRET}
```

Comment:
What will happen if you forget to set the environment variable? Is the error obvious, or does login just mysteriously fail?

Reply:
Error: [CompletionException: org.pac4j.core.exception.TechnicalException: Bad token response, error=unauthorized_client]

Reply:
That's probably good enough for now. If it trips people up, we can put some more checks around it.